.github/workflows/ci.yml

```diff
@@ -2,57 +2,53 @@ name: Cairo-Dojo CI
 
 on:
   push:
-    branches: [ main, master ]
+    branches: [main]
   pull_request:
-    branches: [ main, master ]
+    branches: [main]
+
+env:
+  DOJO_VERSION: 1.6.1
+  SCARB_VERSION: 2.10.1
+  WORKING_DIRECTORY: ./backend/dojo_examples/combat_game
```

Comment:
In the future y'all just need to change the env variable to upgrade the dojo and scarb version